app/assets/stylesheets/welcome.scss

```diff
@@ -21,21 +21,33 @@ $backdrop : #C9D6AE;
 }
 
 .fullWidth {
-   width: 100%;
    margin-left: auto;
    margin-right: auto;
    max-width: initial;
+   width: 100%;
 }
 
 #welcome-title {
-  font-size: 80px;
   color: white;
+  font-size: 80px;
   margin-left: 10px;
-  word-wrap: break-word;
+  width: 560px;
 }
 
 #welcome-description {
-  font-size: 24px;
   color: white;
+  font-size: 24px;
+  margin-left: 10px;
+  width: 600px;
+}
+
+#app-info {
+  background: rgba(204, 204, 204, 0.7);
+  border-radius: 2%;
+  color: black;
+  font-size: 24px;
+  margin-top: 20px;
```

Comment:
Properties should be ordered background, border-radius, color, font-size, margin-left, margin-top, opacity, width